guardian/bonobo
# Update extract script to accept params

## What does this change?

Instead of editing the extract script each time we run it we can just send it some params instead.

## How to test

Check it out and run it. The param validation is pretty basic but protects from _most_ unintentional errors I can think of.

## How can we measure success?

Run it, get a report, check the report. If it looks good, success!

## Have we considered potential risks?

Yes, and nothing immediately obvious occurred to me. It's still pretty fast and access creds are required, so should be minimal impact. 

## Images

N/A

## Accessibility
